### PR TITLE
Added vault template for agora config.

### PR DESCRIPTION
**Pull request by jacarey**

### CODE REVIEW

File: agora.ctmpl [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+swagger {
+    apiDocs = "api-docs"
+    apiVersion = "0.1"
+    baseUrl = "/"
+    contact = "dsde-engineering@broadinstitute.org"
+    description = "Agora stores nethods along with metadata."
+    info = "Agora Methods Repository"
+    license = "BSD"
+    licenseUrl = "http://opensource.org/licenses/BSD-3-Clause"
+    swaggerVersion = "1.2"
```
Comment (bradtaylor) [inline agora.ctmpl]: Swagger version should be 2, yes?

File: agora.ctmpl [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+
+mongodb {
+    host = "{{.Data.mongodb_host}}"
+    port = 27017
+    user = "{{.Data.mongodb_user}}"
+    password = "{{.Data.mongodb_password}}"
+    db = "{{.Data.mongodb_db}}"
+}
+
+environment = "dev"
```
Comment (bradtaylor) [inline agora.ctmpl]: What is this key, and why is it hardcoded "dev"? How does it relate to the environment set at line1?

File: agora.ctmpl (hunk 1 above)
Comment (jacarey) [inline agora.ctmpl]: yup

File: agora.ctmpl [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+modules {
+  kamon-statsd {
+    auto-start = yes
+    requires-aspectj = no
+    extension-id = "kamon.statsd.StatsD"
+    }
+  }
+}
+{{end}}
+{{end}}
```
Comment (bradtaylor) [inline agora.ctmpl]: why {{end}} twice?

File: agora.ctmpl [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+
+webservice {
+    port = "{{.Data.webservice_port}}"
+    interface = "0.0.0.0"
+    host = "{{.Data.webservice_host}}"
+    schema = "https"
+}
+
+instance {
+    name = "reference"
```
Comment (bradtaylor) [inline agora.ctmpl]: Remind me what this is?

File: agora.ctmpl (hunk 2 above)
Comment (jacarey) [inline agora.ctmpl]: likely I can get it from the env set on line one... this also supports "local" which would never be set externally... waiting to talk to Marc about this

**Comment by bradtaylor** [comment]
A couple questions I should probably know the answers to. Otherwise Thumbsup

File: agora.ctmpl (hunk 4 above)
Comment (jacarey) [inline agora.ctmpl]: Useless... that's what it is.

File: agora.ctmpl (hunk 3 above)
Comment (jacarey) [inline agora.ctmpl]: no clue.. we'd have to ask Marc likely closes off the two {{with}} at the top

**Comment by jacarey** [comment]
And of course I made this PR against master.

**Pull request closed without merging**